### DISCUSSION TITLE
The new update has probably broken the plugin.

### DISCUSSION THREAD

**Discussion by txtyash** [nvim-neorg/neorg#254] [Q&A] [answered]
I have used the given sample configuration to configure neorg and in the past it worked just fine but after the update I read the notice and started installing and configuring neorg as given in the docs but it wouldn't work:
```
Error detected while processing BufEnter Autocommands for "*.norg":
E5108: Error executing lua /usr/share/nvim/runtime/lua/vim/treesitter/query.lua:161: query: invalid node type at
position 277
stack traceback:
        [C]: in function '_ts_parse_query'
        /usr/share/nvim/runtime/lua/vim/treesitter/query.lua:161: in function 'get_query'
        /usr/share/nvim/runtime/lua/vim/treesitter/languagetree.lua:37: in function 'new'
        /usr/share/nvim/runtime/lua/vim/treesitter.lua:45: in function '_create_parser'
        /usr/share/nvim/runtime/lua/vim/treesitter.lua:93: in function 'get_parser'
        ...t/neorg/lua/neorg/modules/core/norg/concealer/module.lua:444: in function 'trigger_code_block_highligh
ts'
        ...t/neorg/lua/neorg/modules/core/norg/concealer/module.lua:2078: in function 'on_event'
        ...e/nvim/site/pack/packer/start/neorg/lua/neorg/events.lua:153: in function 'broadcast_event'
        ...art/neorg/lua/neorg/modules/core/autocommands/module.lua:47: in function '_neorg_module_autocommand_tr
iggered'
        [string ":lua"]:1: in main chunk
```

**Comment by max397574** [comment, marked as answer]
Make sure you use the main of the parser branch and have updated.

**Reply by vhyrro** [reply]
Bruh, you beat me by like a second lol

**Comment by vhyrro** [comment]
Yo, did you update your treesitter parser too? `:TSUpdate` is what you gotta run. Make sure both Neorg and the TS parser are pointing to their main branches :)

**Reply by txtyash** [reply]
> Yo, did you update your treesitter parser too? `:TSUpdate` is what you gotta run. Make sure both Neorg and the TS parser are pointing to their main branches :)

`TSUpdate` did the job. But I had already put the norg related parsers in ensure installed so they should've been automatically updated.
